datafusion/common/src/pyarrow.rs

```diff
@@ -94,19 +94,14 @@ mod tests {
                 let python_path: Vec<&str> =
                     locals.get_item("python_path").unwrap().extract().unwrap();
 
-                Err(err).expect(
-                    format!(
-                        "pyarrow not found\nExecutable: {}\nPython path: {:?}\n\
+                panic!("pyarrow not found\nExecutable: {}\nPython path: {:?}\n\
```

Comment:
This error appears to have just been being lost previously, which is probably not what was intended